libnmstate/nm/sriov.py

```diff
@@ -75,5 +165,31 @@ def get_info(device):
         info[Ethernet.SRIOV_SUBTREE] = {
             Ethernet.SRIOV.TOTAL_VFS: sriov_setting.props.total_vfs
         }
+        vfs_config = _get_info_sriov_vfs_config(sriov_setting)
```

Comment:
This point also needs to be mentioned in the commit message.
It is an exception to report the config and not the running state.

Something like: "Current implementation reports the VF/s config state (from NM) and not the kernel state."

Reply:
Done, thanks!